manifests/proxy.pp

```diff
@@ -424,6 +430,8 @@
   $tmpdir                                   = $zabbix::params::proxy_tmpdir,
   $allowroot                                = $zabbix::params::proxy_allowroot,
   $include_dir                              = $zabbix::params::proxy_include,
+  $sslcertlocation_dir                      = $zabbix::params::proxy_sslcertlocation,
+  $sslkeylocation_dir                       = $zabbix::params::proxy_sslkeylocation,
```

Comment:
Can you use `Optional[Stdlib::Absolutepath]`  type here?

Reply:
Will update